design/V1/class-diagram.md

```diff
@@ -0,0 +1,40 @@
+```mermaid
+
+classDiagram
+
+    class Shoulder {
+        -MotorController leadMotor
+        -MotorController followerMotor
+        -Encoder encoder
+        -PIDController pid
+        -double targetAngle
+
+        +getAngle() double
+        +getTarget() double
+        +getIsOnTarget() boolean
+        +setTarget(double target) void
```

Comment:
lets call this angle instead of target to stay consistent with the interface of the Hand
```suggestion
        +setTarget(double angle) void
```